tools/src/main/resources/org/bitcoinj/tools/masternodelist-jack-daniels.json

```diff
@@ -0,0 +1,236 @@
+{
+  "7cc58d7ad7770f637d8653ebab90e220a5bd56eeab0e2ec1d63854738ce1174d-18": {
+    "proTxHash": "ca76907d52eaa05d711e0d479300dcb489a7796244d36bdd5d5a66c48817dfa3",
+    "address": "34.222.40.218:20001",
+    "payee": "yevC84xpeLsm1BfxoRD7w5qjcRV9CPdhvK",
+    "status": "ENABLED",
+    "pospenaltyscore": 0,
+    "lastpaidtime": 1671163604,
+    "lastpaidblock": 78445,
+    "owneraddress": "yXEwNg5f5EozivuZc7CfQkRTensP216SPb",
+    "votingaddress": "yXEwNg5f5EozivuZc7CfQkRTensP216SPb",
+    "collateraladdress": "yiXHNJwRuAXRt8wrwfHb9fbuErqBfSsvpm",
+    "pubkeyoperator": "0bd6e45758bed340da1a039b313e698a2fe4eb32bae483f1438219e6d098b6241ca5009b9d6ad4fb53469a880bc7ad01"
+  },
```

Comment:
files such as this are from the `masternodelist` RPC command and help us create a link between a proTxHash and an outPoint value for each masternode.